scripts/globals/mobskills/nether_blast.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------
+-- Ranged Attack
+-- Deals a ranged attack to a single target.
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/mobskills")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    -- Ranged attack only used when target is out of range
+    if (mob:checkDistance(target) > 2) then
```

Comment:
I believe for a mob to be out of range for auto attack it should be over 5 yalms? And I'm curious: does diabolos use any other abilities if his target is out of range?

Reply:
I wasn't sure if there was a "limit" so I just used the same value as another ranged attack I found.  For now I'm just getting the ability functional so the testers can provide feedback.  I'll likely leave this at 2 for now and when I do a follow up commit for tester feedback I'll make the adjustment.

Reply:
Leave this at 2 because the classification for melee "reach" changes slightly based on the job and mob type. 2 yalms is the safe bet until further testing is able to be completed.